protocols/indexer/contracts/Indexer.sol

```diff
@@ -30,28 +30,21 @@ contract Indexer is IIndexer, Ownable {
   // Token to be used for staking (ERC-20)
   IERC20 public stakeToken;
 
-  // Minimum token amount required for staking
-  uint256 public stakeMinimum;
-
   // Mapping of maker token to taker token to market
   mapping (address => mapping (address => Market)) public markets;
 
   // Mapping of address to timestamp of blacklisting
-  mapping (address => uint256) public blacklist;
```

Comment:
Curious. Why was this a number before?

Reply:
It was a timestamp that is unnecessary because events are emitted.

Reply:
as in an address would only be black listed for X duration?

Reply:
Nope it was just the current timestamp.